deploy/apecloud-mysql/config/mysql-scale-vttablet-config-constraint.cue

```diff
@@ -50,6 +50,15 @@
 	// if this flag is true, vttablet will fail to start if a valid tableacl config does not exist
 	enforce_tableacl_config: bool
 
+    // query server read pool size, connection pool is used by regular queries (non streaming, not in a transaction)
+	queryserver-config-pool-size: int & >=0
```

Comment:
I think underscore must be used here in place of '-', like `queryserver_config_pool_size`. otherwise it won't comply with Cue syntax. Similarly, the tpl needs to be modified because the shell script uses the export command, and hyphens are not valid in shell variables.

Reply:
fixed